ansible/config/linux/docker_dnsmasq_conf

```diff
@@ -0,0 +1,19 @@
+# Set dns forwarder.
```

Comment:
OK, I guess I didn't made my point. No problem, let's see this tomorrow together :+1: 

Anyway the work is great!